frontend/src/domain/use_cases/missions/editMission.js

```diff
@@ -1,8 +1,73 @@
 import { generatePath } from 'react-router'
 
+import { missionsAPI } from '../../../api/missionsAPI'
 import { sideWindowActions } from '../../../features/SideWindow/slice'
 import { sideWindowPaths } from '../../entities/sideWindow'
+import { setError } from '../../shared_slices/Global'
+import { setMissionState } from '../../shared_slices/MissionsState'
+import { setMultiMissionsState } from '../../shared_slices/MultiMissionsState'
 
-export const editMission = missionId => dispatch => {
-  dispatch(sideWindowActions.focusAndGoTo(generatePath(sideWindowPaths.MISSION, { id: missionId })))
+export const editMission = missionId => (dispatch, getState) => {
```

Comment:
J'aime bien le [return early]( https://gomakethings.com/the-early-return-pattern-in-javascript/) quand il y a pas mal de conditions perso.
Aussi, je trouve que sortir dans une fonction (dans le même fichier) un bout de code permet de le documenter, et au final on aboutit à un truc lisible en anglais:
```
if (j'aipasuntruc) {
  alorsjefaisça()

  if (ahouijedoisvérifierça) {
    faisonsça()
  }

  return
}

Sinonjefaisunautretruc()
```
Perso, je me retrouve à pas mal de petites fonctions en bas de mon fichier et ça m'aide à reprendre le code par la suite